sync/fromDisk/assessments.js

```diff
@@ -151,6 +161,8 @@ function getParamsForAssessment(assessmentInfoFile, questionIds) {
             forceMaxPoints: question.forceMaxPoints || false,
             triesPerVariant: question.triesPerVariant || 1,
             gradeRateMinutes: questionGradeRateMinutes,
+            canView: question.canView || null,
+            canSubmit: question.canSubmit || null,
```

Comment:
So, the pattern we've used elsewhere is to define all defaults in JS instead of setting `null` here and checking for null in the syncing sproc. I'd be inclined to do the same here.